src/pybind/mgr/dashboard/controllers/orchestrator.py

```diff
@@ -36,7 +36,7 @@ def get_device_osd_map():
              }
     :rtype: dict
     """
-    result = {}
+    result: dict = {}
```

Comment:
This is PY3 type hinting syntax. Can we use that or should we use `# type: dict` instead? This may be a concern when backporting Octopus PRs to a PY2 based release.

@sebastian-philipp What do you think?

Reply:
Thanks for bringing this up. Orchestrator integration will not be backported, but considering we are encouraged to add typing info in Dashboard (are we?), we should have a guideline to follow.

PY3 type hinting syntax will make backport a nightmare.

Reply:
In today's standup we agree:

- Use the PY3-only syntax if the codes are not going to be backported.
- Stick to the comments way `# type: xxxx` if codes are going to be backported potentially.
- If you don't know, stick to the comments way.